lab1.js

```diff
@@ -77,6 +76,35 @@ var sentence2 = 'Come over here so you can scratch my belly.';
 
 // your code goes here
 
+var sentence1 = 'More food please.';
+var sentence2 = 'Come over here so you can scratch my belly.';
+
+sentence1 = sentence1.split(' ');
+sentence2 = sentence2.split(' ');
+
+for (var i = 0; i < sentence1.length; i++) {
+  if (i === sentence1.length - 1) {
```

Comment:
I like how you used the if block here to get the period on the end of the string.